ocs_ci/framework/conf/default_config.yaml

```diff
@@ -114,3 +114,7 @@ ENV_DATA:
 # This section is related to upgrade
 UPGRADE:
   upgrade_to_latest: true
+  # This default image for ocp upgrade based on GA build
+  ocp_upgrade_version: "4.3.3-x86_64"
```

Comment:
@AviadP @petr-balogh Is there a way to fetch the latest stable z-stream of 4.3 instead of a hardcoded version?

Reply:
I got some info and will implement the function to do that but for now let's keep the version hardcode and then we can add some logic of gather this dynamically.

Reply:
ack

Reply:
I think we can remove here as well and define just channel